src/main/java/capstone/mfslbackend/response/dto/TeamResponse.java

```diff
@@ -10,10 +10,10 @@
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @Data
-@AllArgsConstructor
-@NoArgsConstructor
+//@AllArgsConstructor
+//@NoArgsConstructor
 @ToString
-public class TeamResponse implements Serializable {
+public class TeamResponse extends TeamsResponse implements Serializable {
```

Comment:
Dont know why you made this extend TeamsResponse. TeamsResponse is just a dummy class that holds a list of TeamResponse